daal4py/sklearn/monkeypatch/dispatcher.py

```diff
@@ -145,6 +145,10 @@ def enable(name=None, verbose=True):
     else:
         for key in _get_map_of_algorithms():
             do_patch(key)
+    if deprecation and sys.stderr is not None:
+        sys.stderr.write(
+            "daal4py patching will be deprecated in the future, "
+            "use iskex module instead\n")
```

Comment:
We will need to work on messaging, I prefer to include full name "Intel(R) Extension for Scikit-learn" and provide info how to get it (like pip install intel-sklearn-extension) and how to enable patching.

+ As I remember python has native instruments for deprecation warnings, not just write to stderr.

Reply:
Thanks!
What about 
`daal4py patching will be deprecated in the future, "
            "use Intel(R) Extension for Scikit-learn module instead\n"
            "pip install intel-sklearn-extension\n"
            "python -m iskex <your_script>\n"
            "or\n"
            "from iskex import patch_sklearn\n"
            "patch_sklearn()\n"`
User can disable this warning with patch_sklearn(deprecation=False)

Reply:
```
Scikit-learn patching with daal4py is deprecated and will be removed in the future.
Please, use Intel(R) Extension for Scikit-learn module instead (pip install intel-sklearn-extension)
To enable patching, please, use one of options:
1) python -m iskex <your_script>
2) from iskex import patch_sklearn
   patch_sklearn()
```

Also, please keep in mind, that names of packages are not finalized, so potentially they can be changed.